install-dependencies.sh

```diff
@@ -80,11 +81,23 @@ if [ -z "$DISTRO" ]; then
       DISTRO="mandriva"
    elif [ -f /etc/debian_version ]; then
       DISTRO="debian"
+   elif [ -f /etc/solus-release ]; then
+      DISTRO="solus"
    elif [ -f /usr/local/Homebrew/bin/brew ]; then
       DISTRO="homebrew"
    fi
 fi
 
+asio_not_packaged() {
+   echo "Asio is not packaged for $1, please install it from source:"
+   echo " - Download the latest stable version from"
+   echo "      https://sourceforge.net/projects/asio/files/asio/"
+   echo " - Unpack in your development directory"
+   echo " - Run '$2./configure --prefix=/usr && $2make install-data' in the unpacked asio"
```

Comment:
```suggestion
   echo " - Run './configure --prefix=/usr && $2make install-data' in the unpacked asio"
```

You don't need root for `configure`.